dali/kernels/signal/resampling_gpu.cu

```diff
@@ -77,13 +77,12 @@ void ResamplerGPU<Out, In>::Run(KernelContext &context, const OutListGPU<Out> &o
     desc.out = out_sample.data;
     desc.window = window_gpu_;
     const auto &in_sh = in_sample.shape;
-    const auto &out_sh = out_sample.shape;
```

Comment:
removed this because it was seen as not-used in release builds (therefore a warning)